battleship.py

```diff
@@ -147,17 +147,29 @@ def drawGrid(data, canvas, grid, showShips):
 Returns: bool
 '''
 def isVertical(ship):
-    return
-
+    rows = []
+    for row in range(len(ship)):
+        rows.append(ship[row][0])
+    rows.sort()
+    if rows[0]+1 == rows[1] and rows[1]+1 == rows[2]:
+        return True
+    else:
+        return False
 
 '''
 isHorizontal(ship)
 Parameters: 2D list of ints
 Returns: bool
 '''
 def isHorizontal(ship):
-    return
-
+    cols = []
+    for col in range(len(ship)):
+        cols.append(ship[col][1])
+    cols.sort()
+    if cols[0]+1 == cols[1] and cols[1]+1 == cols[2]:
+        return True
+    else:
+        return False

```

Comment:
@Sunitha101199 
I have no problem with the logic you have to build. Try to re-think and make the code in fewer lines.